.eslintrc.js

```diff
@@ -140,6 +140,13 @@ module.exports = {
       'babel/no-invalid-this': 'off',
       'func-names': 'off',
       'import/order': 'off',
+
+      'no-invalid-this': 'off',
+      'no-unused-expressions': 'off',
+      camelcase: 'off',
```

Comment:
I can't, `quote-props` is set to `as-needed`
We can change it to [`consistent-as-needed`](https://eslint.org/docs/rules/quote-props#options) but it also means adding a key that needs quotes would cause a larger diff in PRs due to needing to update all the other keys in that object

Reply:
Nevermind then, no need to add additional rules that take more time